### DISCUSSION TITLE
Rights needed to run Server Timings?

### DISCUSSION THREAD

**Discussion by Schoettel67** [DaxStudio/DaxStudio#1004] [General]
In our corporate environment something changed and we are no longer able to run Server Timings. Button is active but after “Waiting for Trace to start“ never finishes. Eventually connection is lost. I suspect change in access rights have caused this but having a hard time to ask the right questions internally since I don’t know, what is required. 
anyone else with this problem?

**Comment by marcosqlbi** [comment]
Your account should be an administrator of Analysis Services.

**Reply by Schoettel67** [reply]
Thanks for your answer, Marco. I understand that without admin rights button is de-activated, which it isn’t. 
Additional background: We are using dataset in Power BI service, I’m capacity and workspace admin.

**Reply by marcosqlbi** [reply]
@dgosbell any idea?

**Comment by dgosbell** [comment]
Some VPNs and corporate proxy servers seem to cause issues with the traces for some reason. 

@Schoettel67 - do you know if there have been any recent changes in your organization in this regard? 

Have you tried going into File > Options > Timeouts and extending the trace startup timeout to a larger value to see if that helps?

If you are behind a VPN or Proxy can you try connecting from home or via a mobile hotspot to see if that works? If it does you may need to talk to your network team to see if they can help find a solution.

**Reply by Schoettel67** [reply]
That hit the spot! With VPN disabled, problem is solved. Changing timeout didn’t help, it keeps „Waiting for query end event“. Will have to talk to network guys, then. 
Thanks for your help @dgosbell, @marcosqlbi
